src/app/test_executive/README.md

```diff
@@ -1,9 +1,34 @@
 # How to Run Integration Tests
 
-1) pick an image to run the tests with
-2) build `test_executive.exe` with the `integration_testnet` profile
-3) run test executive, passing in the coda image selected in step 1
-  3.a) it's recommended to run with the `--debug` flag when iterating on tests (this flag will pause testnet cleanup so that you can inspect a testnet after a test fails)
+## Prerequisites
+
+1) ensure the following environment variables are properly set if not already: `GCLOUD_API_KEY`, `KUBE_CONFIG_PATH`, any other vars relating to Google cloud access, vars relating to AWS access, vars relating to ocaml compilation.
```

Comment:
Can you mention that the GCLOUD_API_KEY needs to be set to an api key id for the service account used in step 2?